interface-definitions/service_ntp.xml.in

```diff
@@ -13,6 +13,76 @@
           #include <include/generic-interface.xml.i>
           #include <include/listen-address.xml.i>
           #include <include/interface/vrf.xml.i>
+          <node name="offload">
```

Comment:
I would de-nest these nodes:

Something like:

`set service ntp ptp timestamp interface <name | all>`

The `all` is borrowed from LLDP which does the same - individual interfaces or all